components/merkledb/src/db.rs

```diff
@@ -561,7 +515,7 @@ impl Snapshot for FlushedFork {
     }
 
     fn contains(&self, name: &str, key: &[u8]) -> bool {
```

Comment:
Could it be replace for `self.get(key).is_some()` ?

Reply:
But if use such approach we need a third private method which should return a `Option<&[u8]>` and then in `get` method use clone of the result. It'd slightly complicate the code. So it's better to leave as is, I suppose.

Reply:
We can remove `contains` at all because we have a default implementation that works as your said(`self.get(key).is_some()`). But current implementation of `contains` doesn't require cloning the value.